src/main.lua

```diff
@@ -1961,7 +1908,8 @@ addEventingHandler(ActionMap.Distributions, utils.hasMatchingTag("Action", Actio
 end)
 
 addEventingHandler(ActionMap.ReservedNames, utils.hasMatchingTag("Action", ActionMap.ReservedNames), function(msg)
```

Comment:
Nit: the other paginated handlers have paginated in the name...

Reply:
[c28921d](https://github.com/ar-io/ar-io-network-process/pull/198/commits/c28921d519a53e9cd568164c1215a19c31f085ce)